tests/components/sensor/test_snmp.py

```diff
@@ -0,0 +1,146 @@
+import unittest
+from homeassistant.setup import setup_component
+from tests.common import get_test_home_assistant
+from threading import Thread
+
+from pysnmp.entity import engine, config
+from pysnmp.entity.rfc3413 import cmdrsp, context
+from pysnmp.carrier.asyncore.dgram import udp
+from pysnmp.proto.api import v2c
+from pyasn1.codec.ber import encoder
+from pyasn1.type import univ
+
+import time
+
+_PORT = 10161
+_BASE_OID = (1, 3, 6, 1, 4, 1, 6574, 4)
+_OID1 = _BASE_OID + (1, 1, 1)
+_OID2 = _BASE_OID + (2, 1, 1)
+_OID3 = _BASE_OID + (3, 1, 1)
+
+
+class TestSnmp(unittest.TestCase):
```

Comment:
Please add docstrings to the whole file. Thanks.

Reply:
done